.github/workflows/reusable.rebuild.yml

```diff
@@ -234,12 +236,14 @@ jobs:
       with:
         submodules: true
 
-    - name: Setup MSBuild.exe
-      uses: microsoft/setup-msbuild@v1
+    # We need nmake that wasn't provided by microsoft/msbuild
+    - name: Setup MSVC dev cmd
+      uses: ilammy/msvc-dev-cmd@v1
```

Comment:
How come?

Reply:
See `microsoft/msbuild` repo [issue](https://github.com/microsoft/setup-msbuild/issues/43) and [comment](https://github.com/microsoft/setup-msbuild/issues/58#issuecomment-959027197)

Reply:
I did get it - did you switch to MSBuild or not? If yes, why do you need this script?

Reply:
I switched `nmake` to `msbuild` (though I am a little bit skeptical about using `msbuild` for superbuild but it looks OK in [this](https://github.com/ispc/ispc/actions/runs/6074148124/job/16485316208) run). But for all external projects, `ninja` is still used (to build LLVM, dependencies and ISPC itself).

`microsoft/msbuild` provides only `msbuild` in `PATH`, so I use `ilammy/msvc-dev-cmd` to obtain `ninja`.

Reply:
Can we avoid 3rd party actions? I understand the convenience, but it's a security problem.

Reply:
Yes, I've removed it.